docs/guide/deploy.md

```diff
@@ -182,12 +182,15 @@ Don't enable options like _Auto Minify_ for HTML code. It will remove comments f
          name: github-pages
          url: ${{ steps.deployment.outputs.page_url }}
        needs: build
+       permissions:
+         pages: write
+         id-token: write
```

Comment:
These are not needed at here, they are already there at workflow-level.

Reply:
I have removed them, thanks for the quick feedback.